### PR TITLE
Some proposed changes to rpp-charter.md

### PR DESCRIPTION
**Pull request by billwuqin**
Fix the following several thing
1. Web layer network element is not the key merit of RESTFUL Design
2. Make a clear distinguish between EPP developd by REGEXT WG and RPP developed by this new proposed WG
3. Deliverables part is not consistent with milestone part
4. Need to separate what is byond scope of this charter as independent section.

### CODE REVIEW

File: rpp-charter.md [hunk 1]
```diff
@@ -1,45 +1,26 @@
-# Background and Problem Space
+# Background and Motivation
+The Extensible Provisioning Protocol (EPP) was standardized (STD69) over 20 years ago to address the needs of domain name management between domain name registries and registrars. It encodes protocol Request and response using XML and can be carried over multiple transport protocols. Though EPP has served the domain name industry well, there is now a desire to modernize provisioning protocol using a RESTFUL design and JSON encoding. Such design could take advantage of stateless architecture and widely deployed solutions such as OpenAPI with documentation, testing and code generation tools,L4-L7 network services.
```
Comment (anewton1998) [inline rpp-charter.md:2]: We probably need discussion about allowing the scope to encompass things like OpenAPI, JSON Schema, etc... But this is a good point.

**Review by anewton1998 — COMMENTED** [review]
I tend to agree wtih Eric's comments on the list regarding this PR. It reworks a lot of the charter without much benefit.

File: rpp-charter.md [hunk 2]
```diff
@@ -1,45 +1,26 @@
...
+For example, this evolution has already started in the ccTLD space and first production deployments of this approach have already been adopted by both existing registrars and new registrars as a preferred way of integration for the latter one. A REST architecture may allow easier integration between registries and registrars, thus lowering the costs for domain registration.
 
 # Objective and Scope
+The RPP Working Group is chartered to develop a series of specifications to be known collectively as RESTful Provisioning Protocol (RPP). These specifications will specifically target a REST architecture using HTTPS and JSON. The Working Group will concentrate on the following:
+o Collecting and documenting a set of core requirements for RPP. These requirements will be publicly available on the working group's wiki.
+o Developing RPP Architecture for the provisioning of domain names. It Supports RPP transport over HTTPS and Encode protocol Request and response with JSON.
+o Providing RPP Domain Name JSON Mapping for the different objects to be provisioned.
```
[the remark this answers is not in the document]
Comment (billwuqin) [reply rpp-charter.md:11]: You will see tasklist and milestone list will align with objecrtive list in the charter at the end. For WG charter, the best pratice is to summarize objectives with a few bullets.

File: rpp-charter.md [hunk 3]
```diff
@@ -1,45 +1,26 @@
...
 
-# Milestones
+This working group will find consensus on a set of core requirements for RPP. In alignment with the IESG's Statement on Support Documents dated 2023-08-24, these requirements will be publicly available on the working group's wiki. Once established, these requirements may be changed through explicit working group consensus and in consultation with the group’s responsible Area Director.
 
-After finding consensus regarding requirements, the first document to be 
-delivered to the IESG by this working group will describe the core architecture of RPP with regard to the
-provisioning of domain names, followed by any specifications necessary
-to implement the core architecture and the JSON mapping for the different objects to be provisioned.
+# Dependencies and Liaisons
+The RPP Working Group will closely collaborate with REGEXT WG that addresses topics related to EPP and RDAP use.
```
[the remark this answers is not in the document]
Comment (billwuqin) [reply rpp-charter.md:26]: Okay.

File: rpp-charter.md [hunk 4]
```diff
@@ -1,45 +1,26 @@
...
+Industry experience and use cases from EPP may influence the outputs of this working group, but direct compatibility of RPP with EPP is not a goal. Likewise, extensions or other changes for EPP are out of scope of this working group.
 
-This working group will find consensus on a set of core requirements for RPP.
-In alignment with the IESG's Statement on Support Documents dated 2023-08-24,
-these requirements will be publicly available on the working group's wiki.
-Once established, these requirements may be changed through explicit working group
-consensus and in consultation with the group’s responsible Area Director.
+# Work items
+The RPP Working Group will initially focus on the following deliverables:
+o wiki or internet draft for core requirements for RPP.
```
[the remark this answers is not in the document]
Comment (billwuqin) [reply rpp-charter.md:18]: See IESG statement on support document, https://datatracker.ietf.org/doc/statement-iesg-support-documents-in-ietf-working-groups-20230824/, AD will make final decision whether requirement should be published. So the change we can  tweak is:
OLD TEXT:
"o wiki or internet draft for core requirements for RPP."
NEW TEXT:
"document core requirements for RPP."
but in the milestone section, you don't add it as one milestone, which will distinct from other work items that MUST be delivered.

File: rpp-charter.md (hunk 1 above)
Comment (billwuqin) [reply rpp-charter.md:2]: Yes, the change proposed here relfects the conclusion of discussion with Pawel Kowalik.

File: rpp-charter.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (anewton1998) [reply rpp-charter.md:11]: As the AD has already preliminarily looked at the charter and has not mentioned this issue, I am inclined to leave the objectives as a general summary.

File: rpp-charter.md (hunk 4 above)
[the remark this answers is not in the document]
Comment (anewton1998) [reply rpp-charter.md:18]: That is not my reading of the IESG statement. However, the AD has done a preliminary review and has not called this issue out. If he changes his mind later, he will let us know. For now I am inclined to leave this as publication on the wiki.

File: rpp-charter.md (hunk 1 above)
Comment (anewton1998) [reply rpp-charter.md:2]: I believe this has been covered in another PR.

**Comment by anewton1998** [comment]
I am closing this PR because either all these items are covered in other PRs or they are changes of reorganization of the charter that have yet been raised by the AD.

**Pull request closed without merging**